purchase_order_maintenance_equipment_link/models/maintenance_equipment.py

```diff
@@ -0,0 +1,38 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields, _
+
+
+class MaintenanceEquipment(models.Model):
+    _inherit = 'maintenance.equipment'
+
+    count_purchase_order_ids = fields.Integer(compute='_compute_purchase_orders')
```

Comment:
Please rename this field, because any field finished in `_ids` should be a `Many2one` or `Many2many` field. Usual convention for this:
```suggestion
    purchase_order_count = fields.Integer(compute='_compute_purchase_orders')
```